ossdl-cdn.php

```diff
@@ -272,13 +272,44 @@ function scossdl_off_options() {
 	$example_cdn_uri .= '/wp-includes/js/jquery/jquery-migrate.js';
 	$example_cdn_uri  = esc_url( $example_cdn_uri );
 	?>
+		<h3><?php _e( 'Simple CDN' ); ?></h3>
 		<p><?php _e( 'Your website probably uses lots of static files. Image, Javascript and CSS files are usually static files that could just as easily be served from another site or CDN. Therefore, this plugin replaces any links in the <code>wp-content</code> and <code>wp-includes</code> directories (except for PHP files) on your site with the URL you provide below. That way you can either copy all the static content to a dedicated host or mirror the files to a CDN by <a href="https://knowledgelayer.softlayer.com/faq/how-does-origin-pull-work" target="_blank">origin pull</a>.', 'wp-super-cache' ); ?></p>
 		<p><?php printf( __( '<strong style="color: red">WARNING:</strong> Test some static urls e.g., %s  to ensure your CDN service is fully working before saving changes.', 'wp-super-cache' ), '<code>' . esc_html( $example_cdn_uri ) . '</code>' ); ?></p>
 
 	<?php if ( get_home_url() !== $ossdl_off_blog_url ) { ?>
 		<p><?php printf( __( '<strong style="color: red">WARNING:</strong> Your siteurl and homeurl are different. The plugin is using %s as the homepage URL of your site but if that is wrong please use the filter "ossdl_off_blog_url" to fix it.', 'wp-super-cache' ), '<code>' . esc_html( $ossdl_off_blog_url ) . '</code>' ); ?></p>
 	<?php } ?>
-
+		<h3><?php _e( 'Jetpack CDN' ); ?></h3>
+		<p><?php printf(
+			__( 'The free %1$sJetpack plugin%2$s has a %3$sSite Accelerator%2$s feature that is better than the CDN in this plugin as it will offload requests for static files off your server. ', 'wp-super-cache' ),
```

Comment:
`that is better than the CDN in this plugin` -> `that is may be preferable to many traditional CDN services offered in this plugin`

Reply:
I thought of this as WordPress 5.0 shipped but from [JetPack's docs](https://jetpack.com/support/site-accelerator/) 

> No cache invalidations – currently the images are cached “forever” and static assets will be tied >to the public version of WordPress, Jetpack, or WooCommerce that you’re using. For images, if >you want to “refresh” an image you will need to change its file name. Adding random query >arguments, commonly known as cachebusters, will not work.

That's a serious limitation to bear in mind while recommending it to more than a million installs of WPSC. 
Otherwise, this is a great idea

Reply:
That's a really good point. Maybe we should try to make the callout a little less "you should do it" and more "here's another (potentially easier) option"

Reply:
I agree. The wording on the CDN page should make reference to this limitation of Site Accelerator. I'll update it.

Reply:
I updated the language, and also disabled the CDN if SA is active in https://github.com/Automattic/wp-super-cache/pull/636/commits/0b641bf8407ad78adcc7a0a46e98ab6d529817e9